.github/actions/perform-static-analysis/action.yaml

```diff
@@ -1,8 +1,14 @@
 name: "Perform static analysis"
 description: "Perform static analysis"
 inputs:
+  sonar_organisation_key:
+    description: "Sonar organisation key, used to identify the project"
+    required: false
+  sonar_project_key:
+    description: "Sonar project key, used to identify the project"
+    required: false
   sonar_token:
-    description: "Sonar token API key"
+    description: "Sonar token, the API key"
     required: false
```

Comment:
Could the `secrets:` key be used here instead? I'm not too sure on the differences but I believe it helps GitHub in identifying the value as secret and obscuring it in outputs etc

Reply:
Good question! As this is a composite action it appears that even a secret has to be passed as an input. However, it is still reduced from the output as the calling workflow is aware it shouldn't be display in plaintext:

![image](https://github.com/nhs-england-tools/repository-template/assets/499338/cfc022fc-613a-41c6-a437-0ceb941da99a)

Reply:
Makes sense!